manageiq-operator/Dockerfile

```diff
@@ -1,5 +1,18 @@
 # Build the manager binary
-FROM docker.io/library/golang:1.23 as builder
+FROM registry.access.redhat.com/ubi9/ubi-minimal as builder
+
+COPY go.mod /go.mod
```

Comment:
This is used to detect the desired Go major.minor version, but could be changed to an ENV var if we want to extract this to a script for reuse.